proposals/stage-2/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md

```diff
@@ -1,55 +1,49 @@
-# Project Name
+# ${Project Name}
 
 ## Project Champion
 
 Who is the main point of contact during the application process?
 
-## CPC Champion (optional)
-
-If you have a champion from the CPC please identify them. If you do not currently have one we will identify someone to support your project.
-
 ## Description
 
-A rough description of the project
+A rough description of the project in less than 100 words.
 
 ## Statement of alignment with OpenJS Foundation charter and mission
 
-Please refer to [TBD][]
-
-## Current Code of Conduct
+Please refer to [the Cross Project Council's Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md).
+Keep answers to less than 250 words.
 
-Include a link
+## List of all repos that are part of the project
 
-## Current Project License
-
-Include a link
-
-## Current Source Control
-
-Link to Github or other source control
+Include links and license information
 
 ## Issues Tracker
 
 Link to GitHub or other tracker
 
-## External Dependencies
+## Current Code of Conduct
 
-Full list of dependencies in your source tree including licenses.
+Include a link
 
-## Release Methodology and mechanics
+## Vendored Dependencies (Optional)
 
-How often do you release and where are the releases made. Do you have an LTS program?
+Full list of dependencies in the source tree of all repos. Please include licenses if you can.
+A full audit of all vendored dependencies will be required before your project can officially
+join the foundation, but this can be done later with foundation support.
 
-## List of Committers
+## List of Committers (Optional)
 
-Full list of individuals with a commit bit to your project and their corporate affiliation if applicable.
+Full list of individuals with commit access to your project repos and their corporate affiliation
+if applicable. A full audit of all collaborators will be required before your project can officially
+join the foundation, but this can be done later with foundation support.
```

Comment:
I don't think we have ever done this for Node.js organization. We are not requiring this step for any new collaborators joining Node.js. This is going to create issues for Node.js, as we have some individuals that are not contributing with their full name.

Is this needed? What's the reasoning behind this change?

Reply:
Seconding the concerns expressed by @tobie and @mcollina, this "full audit of all collaborators" seems like a rather surprising thing to ask for, both in its vagueness and its potential scope.

Reply:
I think the intent here was to gather data for staging purposes (at least, that is what I was intending/interpreting) because we have a requirement for Impact projects to have 'open governance' as defined by a percentage of committers from outside a single company iirc. Perhaps the word 'Audit' is the wrong word to be using. listing, maybe?

Reply:
I've removed this section